zenodo/modules/records/templates/zenodo_records/macros/admin.html

```diff
@@ -67,7 +91,14 @@
 {%- endmacro %}
 
 {%- macro safelist_action(user, action) %}
-  {{user is safelisted_user}}
+  {% if user.safelist %}
+    {% set safelister = user | safelisted_by %}
+    {% set safelisted_on = user.safelist[0].created.date() %}
```

Comment:
```suggestion
    {% set safelisted_on = user.safelist.created.date() %}
```

Piggy-backing on https://github.com/zenodo/zenodo/pull/2413#discussion_r1071272100, this would be possible if you do something [here](https://github.com/zenodo/zenodo/blob/b87a52f58679976d38a53c93e2425999e469e4f7/zenodo/modules/spam/models.py#L60) like:

```python
    user = db.relationship(User, backref='safelist', uselist=False)
```

Reply:
This didn't work for me for some reason, let's discuss IRL when you have a chance.